templates/menu.html

```diff
@@ -34,6 +34,9 @@
         <div class="col-sm">
             <a href="{{ url_for('addTeam', session_id=session_id)}}"><button class="my-btn">Add Team</button></a>
         </div>
+        <div class="col-sm"> <!-- Next 3 lines are cclevinger's -->
+            <a href="{{ url_for('addTeamCSV', session_id=session_id)}}"><button class="my-btn">Import Students</button></a>
```

Comment:
It looks like we could pass in a session id to to the addTeamCSV page from here. You could make it a get parameter.